packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -70,7 +70,7 @@ export const TabsItem = compose<TabsItemType>({
       },
     };
 
-    const buttonRef = useViewCommandFocus(componentRef);
+    const buttonRef = Platform.OS !== 'macos' ?  useViewCommandFocus(componentRef) : componentRef;
```

Comment:
I think we have an open bug to make UseViewCommandFocus work on macOS. Once we do so, I think you can undo the platform check here

Reply:
But also, what does useViewCommandFocus do here?

Reply:
As of #919 , useViewCommandFocus is implemented for macOS. Try removing this platform check and let me know if it works! =)

Reply:
It works now, thank you!

Reply:
update: useViewCommandFocus doesn't seem to be working when tested again, talked offline about this and will be filing a bug